trigger_scripts/jrap_prep.wc2.pbs

```diff
@@ -0,0 +1,144 @@
+module load intel/${intel_ver}
+module load craype/${craype_ver}
+module load cray-mpich/${cray_mpich_ver}
+module load cray-pals/${cray_pals_ver}
+module load hdf5/${hdf5_ver}
+module load netcdf/${netcdf_ver}
+module load cfp/${cfp_ver}
+
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)prod_util/") -eq 0 ]]; then echo "prod_util is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)prod_envir/") -eq 0 ]]; then echo "prod_envir is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)util_shared/") -eq 0 ]]; then echo "util_shared is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)grib_util/") -eq 0 ]]; then echo "grib_util is not loaded!"; fi
+
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)hdf5/") -eq 0 ]]; then echo "hdf5 is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)netcdf/") -eq 0 ]]; then echo "netcdf is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)w3nco/") -eq 0 ]]; then echo "w3nco is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)w3emc/") -eq 0 ]]; then echo "w3emc is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)bacio/") -eq 0 ]]; then echo "bacio is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)bufr_dump/") -eq 0 ]]; then echo "bufr_dump is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)bufr/") -eq 0 ]]; then echo "bufr is not loaded!"; fi
+
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)envvar/") -eq 0 ]]; then echo "envvar is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)PrgEnv-intel/") -eq 0 ]]; then echo "PrgEnv-intel is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)intel/") -eq 0 ]]; then echo "intel is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)craype/") -eq 0 ]]; then echo "craype is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)cray-mpich/") -eq 0 ]]; then echo "cray-mpich is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)cray-pals/") -eq 0 ]]; then echo "cray-pals is not loaded!"; fi
+if [[ $(echo $LOADEDMODULES | egrep -c "(^|:)cfp/") -eq 0 ]]; then echo "cfp is not loaded!"; fi
+
+
+#export NPROCS=3
+
+export SENDECF=NO  # developer
+export SENDSDM=NO  # developer
+#export SENDDBN=YES; export DBNROOT="echo test " # developer
+
+export DEBUG_LEVEL=3
+
+export DATAROOT=/lfs/h2/emc/ptmp/$USER
+export jlogfile=/lfs/h2/emc/stmp/$USER/${JTYP}_prep.${PDY}.jlogfile
+
+export COMIN_ROOT=/lfs/h2/emc/ptmp/$USER/com
+export tstsp=/lfs/h2/emc/ptmp/$USER/com/obsproc/${JTYP}.${PDY}/${JTYP}.t${cyc}z.
+
+export COMINgdas=/lfs/h1/ops/canned/com/gfs/${gfs_ver}/gdas.${PDY}/${cyc}/atmos
+export COMINgfs=/lfs/h1/ops/canned/com/gfs/${gfs_ver}/gfs.${PDY}/${cyc}/atmos
+export GETGES_COM=/lfs/h1/ops/canned/com/gfs/${gfs_ver}/
+export GESROOT=/lfs/h1/ops/canned/com/gfs/${gfs_ver}/
+
+export COMOUT_ROOT=/lfs/h2/emc/ptmp/$USER/com/
+
+export KEEPDATA=YES
+export POE=NO
+export BACK=NO
+export NSPLIT=1
```

Comment:
Let's remove or comment these lines for the NCO trigger. These were required for WCOSS Cray where obsproc_rap is currently running in operations. These settings were required in order to force rap_prep to run in serial after the upgrade of GFS to v16. This may not be a requirement for WCOSS2 (*fingers crossed*).